packages/e2e-tests/src/config/bootstrap.js

```diff
@@ -101,6 +101,11 @@ const ALLOWED_ERROR_MESSAGES = [
 
   // See https://www.chromestatus.com/feature/508239670987980
   "Blocked attempt to show a 'beforeunload' confirmation panel for a frame that never had a user gesture since its load",
+
+  // See https://github.com/GoogleForCreators/web-stories-wp/pull/11782#issuecomment-1178865825
+  'Bundle not found for language en:',
```

Comment:
The link does not really provide any additional value. Let's do something like this instead.

```suggestion
  // Sometimes the AMP viewer can fail to load translations when viewing a story.
  'Bundle not found for language en:',
```